modules/stochastic_tools/test/tests/multiapps/commandline_control/master_multiple.i

```diff
@@ -0,0 +1,49 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+[]
+
+[Problem]
+  type = FEProblem
+  solve = false
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Distributions]
+  [./uniform]
+    type = UniformDistribution
+    lower_bound = 5
+    upper_bound = 10
+  [../]
+[]
+
+[Samplers]
+  [./sample]
+    type = MonteCarloSampler
+    n_samples = 3
+    distributions = 'uniform uniform'
+    execute_on = 'pre_initial'
+  [../]
+[]
+
+[MultiApps]
+  [sub]
+    type = FullSolveMultiApp
+    positions = '0 0 0
```

Comment:
All of these tests specify `positions`, but it seems like you wouldn't want to do that for the standard use case. You don't have to specify `positions`, do you? If not, I would suggest setting these up without that.

Reply:
This is just a means to create multiple sub applications, this is the only way to have multiple sub-apps that I know about.

Reply:
So if you want to run a million realizations of the sampled variables, you need a million positions listed?  That's not going to work very well.

Reply:
Agreed, but that is a problem with FullSolveMultiApp, not with this issue.

Reply:
Sure.  Maybe I wasn't using FullSolveMultiApp before, because I don't recall needing to do this when I used sampling with controls before.

Reply:
Looks like I was using SamplerMultiApp.

Reply:
You probably used the "SamplerMultiApp", which figures this out for you. That might work here as well, but I didn't want to make the test as minimal as possible, w/r/t stocastic_tools objects.

Reply:
I can change this test if you prefer.

Reply:
I think it's fine as-is. There's some more work to do in this area to sort out the best way to run our problems of interest, but that's for later.